css/site.styl

```diff
@@ -27,21 +27,27 @@ normalize()
 
 // VARIABLES
 
-$color-black-soft = rgb(14, 14, 14)
-$color-white-soft = rgb(253, 253, 253)
+$color-black = rgb(0, 0, 0)
+$color-white = rgb(255, 255, 255)
 
 // ******************************************************
 
 // BASIC
 
 html
-  background-color: $color-white-soft
-  color: $color-black-soft
+  background-color: $color-white
+  color: $color-black
 
 img
   max-width: 100%
   height: auto
 
+.mobile-only
+  display: initial !important 
```

Comment:
i ran into an issue where i was trying to make something mobile-only, but it was being overridden by a utility class on its parent. it was a `ul.u-inline-list li` and i wanted one of the `<li>`s to be mobile-only but it was overridden by that parent class.  i think this is a good use of `! important`